types/micromatch/index.d.ts

```diff
@@ -1,171 +1,621 @@
-// Type definitions for micromatch 2.3.7
-// Project: https://github.com/jonschlinkert/micromatch
+// Type definitions for micromatch 3.1
```

Comment:
If you want to update the package to a new major version, you should follow the guide: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#i-want-to-update-a-package-to-a-new-major-version

Reply:
How should I go about restoring the previous version, should I edit the commit, or just add the original files in a new commit?

Reply:
I think adding them as a new commit is good to go.